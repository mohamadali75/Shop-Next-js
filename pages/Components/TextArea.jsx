import React from "react";
import { css } from "@emotion/css";
import { useTheme } from "@mui/material";



function TextArea({placeholder}) {
    const theme=useTheme()
    return(
        <textarea cols="30" rows="4" className={css`
        border: solid ${theme.palette.primary.border};
        
   }
   }
 `}  placeholder={placeholder}/>
    )
}
export default TextArea
