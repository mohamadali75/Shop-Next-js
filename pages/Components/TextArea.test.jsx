import renderer from 'react-test-renderer';
import React from "react"
import TextArea from './TextArea';

it('textarea test', () => {
  const component = renderer.create(
    <TextArea>test</TextArea>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();})
