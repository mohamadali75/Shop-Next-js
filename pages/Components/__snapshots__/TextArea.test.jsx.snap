// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`textarea test 1`] = `
<textarea
  className="css-10s24d6"
  cols="30"
  rows="4"
/>
`;
